### PR TITLE
Add org.gnome.gitlab.ilhooq.Bookup

### PR DESCRIPTION
**Pull request by ilhooq**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission

### CODE REVIEW

File: org.gnome.Bookup.json [hunk 1]
```diff
@@ -0,0 +1,68 @@
+{
+    "app-id" : "org.gnome.Bookup",
```
[the remark this answers is not in the document]
Comment (hfiguiere) [reply org.gnome.Bookup.json:2]: `org.gnome.gitlab`.USERNAME.APPNAME is available.

File: org.gnome.gitlab.ilhooq.Bookup.json [hunk 2]
```diff
@@ -60,7 +60,7 @@
                     "type" : "git",
                     "url" : "https://gitlab.gnome.org/ilhooq/bookup.git",
                     "tag" : "v0.1.0",
-                    "commit": "62e17a0ca070e613cabc88a4b09f17ed1d198636"
+                    "commit": "4475d66c643eb5654f5da392f842ce1016224d24"
```
Comment (hfiguiere) [inline org.gnome.gitlab.ilhooq.Bookup.json:63]: retagging is a bad practice.

**Comment by hfiguiere** [comment]
bot, build org.gnome.gitlab.ilhooq.Bookup

File: org.gnome.gitlab.ilhooq.Bookup.json (hunk 2 above)
Comment (ilhooq) [reply org.gnome.gitlab.ilhooq.Bookup.json:63]: I have to change the tag or it's ok ?

Comment (hfiguiere) [reply org.gnome.gitlab.ilhooq.Bookup.json:63]: in general you never retag a published tag. It's bad practice. this is not flathub specific.

**Pull request closed without merging**